WordPress/Docs/CodeAnalysis/AssignmentInTernaryConditionStandard.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd" 
+    title="Assignment In Ternary Condition"
+    >
+    <standard>
+    <![CDATA[
+    Variables should not be assigned in conditional statement of ternary. 
+    Condition must be in parentheses to be checked; ternaries lacking parentheses around condition are skipped. 
```

Comment:
```suggestion
    The condition must be in parentheses to be checked; ternaries lacking parentheses around the condition are skipped. 
```

Similar to the above, the sentence reads better to me if we add the article "the", but feel free to ignore this comment if that is not the case.